### PR TITLE
add week 4 blog leon

### PR DESCRIPTION
**Pull request by llu0**
somehow i pushed directly to main at first...

### CODE REVIEW

**Comment by harvey-fin** [comment]
Great progress. I am particularly interested in the activation patching part where patching from middle-injected leads to higher outcome logits. Is it correlated with "higher-attention to the adversarial prompt -> larger V-vector extracted and added to the residual stream -> more successful activation patching"? Excited to read more about this.
